### PR TITLE
Update README with build instructions

### PR DESCRIPTION
**Pull request by paulcastro**
This PR adds simple build instructions to the README.  I'm submitting this for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+`wskdeploy` is a utility to help you create and deploy OpenWhisk projects. Deploy all your actions, triggers, and rules using a single command! You can use this in addition to the OpenWhisk CLI.
+
+`wskdeploy` is currenty under development and in its very early stages.  Check back often to see how its progressing.
+
+# How to Build
+`wskdeploy` is written in Go.
+
+1. Setup your [Go development environment](https://golang.org/doc/code.html).
+
+2.  `wskdeploy` depends on the `github.com/openwhisk-client-go/whisk` . To install:
```
Comment (daisy-ycguo) [inline README.md]: `github.com/openwhisk-client-go/whisk` should be `github.com/openwhisk/openwhisk-client-go/whisk`

**Comment by daisy-ycguo** [comment]
Addressed issue #25

File: README.md (hunk 1 above)
Comment (paulcastro) [inline README.md]: Corrected and amended commit.

**Comment by daisy-ycguo** [comment]
Merged.

**Pull request closed without merging**